Rx.playground/Pages/Playground.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,22 @@
+/*:
+ > # IMPORTANT: To use **Rx.playground**:
+ 1. Open **Rx.xcworkspace**.
+ 1. Build the **RxSwift-macOS** scheme (**Product** → **Build**).
+ 1. Open **Rx** playground in the **Project navigator**.
+ 1. Show the Debug Area (**View** → **Debug Area** → **Show Debug Area**).
+ */
+import RxSwift
+/*:
+ # Try Yourself
+ 
+ It's time to play with Rx 🎉
+ */
+playgroundShouldContinueIndefinitely()
+
+example("Try yourself") {
+  let disposeBag = DisposeBag()
```

Comment:
Hi @devxoul ,

I think we can just write

```
// let disposeBag = DisposeBag()
_ = Observable.just("Hello, RxSwift!")
  .debug("Observable")
  .subscribe()
  // If dispose bag is used instead, sequence will terminate on scope exit
  // .disposed(by: disposeBag)
```

... or some comment worded better.